crates/bevy_window/src/window.rs

```diff
@@ -133,9 +140,9 @@ pub struct Window {
     /// macOS X transparent works with winit out of the box, so this issue might be related to: <https://github.com/gfx-rs/wgpu/issues/687>.
     /// You should also set the window `composite_alpha_mode` to `CompositeAlphaMode::PostMultiplied`.
     pub transparent: bool,
-    /// Should the window start focused?
+    /// Should the window be focused?
```

Comment:
Are you sure this isn't "is this window currently focused?"

Reply:
Not sure what you mean. "is this window currently focused" implies it's only a "getter" to me, like you can't be used to set the focus, `should` reflects both

Reply:
Ah: I read it as only a setter. We should experiment with other phrasing to make sure both uses are clear.

Reply:
Maybe resolved by https://github.com/bevyengine/bevy/pull/8858/commits/6231a52698480e9210df27c50c0401f8ae0a7771 with @SpecificProtagonist 's proposed change